cmd/iconbridge/stat/stat_test.go

```diff
@@ -2,33 +2,13 @@ package stat
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
 	"testing"
 	"time"
 
 	"github.com/icon-project/icon-bridge/common/log"
 )
 
-func TestSystemMetricsGetter(t *testing.T) {
```

Comment:
Are the tests on stat_test.go removed because they do not work on MacOS ? @andrii-kl

Reply:
@manishbista28  I have removed it because it fails and it is not a unit test. 
This test is running the function that try to find some files on the system.
Such behavior should be mocked.